pkg/commands/git_commands/branch_loader.go

```diff
@@ -122,11 +130,74 @@ func (self *BranchLoader) Load(reflogCommits []*models.Commit) ([]*models.Branch
 			branch.UpstreamRemote = match.Remote
 			branch.UpstreamBranch = match.Merge.Short()
 		}
+
+		// If the branch already existed, take over its BehindBaseBranch value
+		// to reduce flicker
+		if oldBranch, found := lo.Find(oldBranches, func(b *models.Branch) bool {
+			return b.Name == branch.Name
+		}); found {
+			branch.BehindBaseBranch.Store(oldBranch.BehindBaseBranch.Load())
+		}
+	}
+
+	if loadBehindCounts && self.UserConfig.Gui.ShowDivergenceFromBaseBranch != "none" {
+		onWorker(func() error {
+			return self.GetBehindBaseBranchValuesForAllBranches(branches, existingMainBranches, renderFunc)
+		})
 	}
 
 	return branches, nil
 }
 
+func (self *BranchLoader) GetBehindBaseBranchValuesForAllBranches(
+	branches []*models.Branch,
+	existingMainBranches *ExistingMainBranches,
+	renderFunc func(),
+) error {
+	mainBranches := existingMainBranches.Get()
+	if len(mainBranches) == 0 {
+		return nil
+	}
+
+	t := time.Now()
+	errg := errgroup.Group{}
```

Comment:
I decided to remove the SetLimit(10) call here again; it no longer seems to be necessary, now that we delay this until after the branches have been loaded the second time. Let me know if you disagree.